src/graphql/typeDefs/mutations/partners.graphql

```diff
@@ -1,5 +1,5 @@
 type PartnersMutation {
-  create(partner: PartnerCreateInput!): Partner!
-  delete(id: ID!): ID
-  partner(id: ID!): PartnerMutation
+  create(partner: PartnerCreateInput!): Partner! @auth(requires: "partners")
```

Comment:
Creating a partner isn't admin only? Does this mean a partner with `partner` scope may be able to create a new partner record?

Reply:
it does.. although for now we can just keep that scope to the admins. let's talk through what we really want the scopes to be longer term as we start to see how this works with sessions.